.azure-pipelines-templates/install_extended_testing_tools.yml

```
@@ -0,0 +1,8 @@
+steps:
+  - script: |
+      set -ex
+      sudo apt-get -y update
+      sudo apt install ansible -y
+      cd getting_started/setup_vm
+      ansible-playbook ccf-extended-testing.yml
+    displayName: Install Extended Testing Tools
```

```suggestion
      ./run.sh ccf-extended-testing.yml
    workingDirectory: getting_started/setup_vm
    displayName: Install Extended Testing Tools
```

Slightly neater I think.